services/s3_gate/cfg/config.yml

```diff
@@ -17,11 +17,13 @@ max_clients_count: 100
 max_clients_deadline: 30s
 
 server:
-  - tls:
+  - address: 0.0.0.0:8080
```

Comment:
Isn't there some default for it?

Reply:
> Isn't there some default for it?

Hmm?
Can you please explain it in more detail?

Reply:
OK, there is no default, forget about it.